index.html

```diff
@@ -1,13 +1,307 @@
+            >
+              <path
+                d="M0 .5A.5.5 0 0 1 .5 0h15a.5.5 0 0 1 .5.5v3a.5.5 0 0 1-.5.5H14v2h1.5a.5.5 0 0 1 .5.5v3a.5.5 0 0 1-.5.5H14v2h1.5a.5.5 0 0 1 .5.5v3a.5.5 0 0 1-.5.5H.5a.5.5 0 0 1-.5-.5v-3a.5.5 0 0 1 .5-.5H2v-2H.5a.5.5 0 0 1-.5-.5v-3A.5.5 0 0 1 .5 6H2V4H.5a.5.5 0 0 1-.5-.5v-3zM3 4v2h4.5V4H3zm5.5 0v2H13V4H8.5zM3 10v2h4.5v-2H3zm5.5 0v2H13v-2H8.5zM1 1v2h3.5V1H1zm4.5 0v2h5V1h-5zm6 0v2H15V1h-3.5zM1 7v2h3.5V7H1zm4.5 0v2h5V7h-5zm6 0v2H15V7h-3.5zM1 13v2h3.5v-2H1zm4.5 0v2h5v-2h-5zm6 0v2H15v-2h-3.5z"
+              />
+            </svg>
+            <h3 class="session-title">Breaking Political Walls</h3>
+            <p class="session-content">
+              Participants will get a chance to connect and network throughout
+              the two days event. This will be a great opportunity build
+              relations between opposing political parts in Iraq.
+            </p>
+          </div>
+          <div class="program-sessions">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="50"
+              height="50"
+              fill="currentColor"
+              class="bi bi-clock-history"
+              viewBox="0 0 16 16"
+            >
+              <path
+                d="M8.515 1.019A7 7 0 0 0 8 1V0a8 8 0 0 1 .589.022l-.074.997zm2.004.45a7.003 7.003 0 0 0-.985-.299l.219-.976c.383.086.76.2 1.126.342l-.36.933zm1.37.71a7.01 7.01 0 0 0-.439-.27l.493-.87a8.025 8.025 0 0 1 .979.654l-.615.789a6.996 6.996 0 0 0-.418-.302zm1.834 1.79a6.99 6.99 0 0 0-.653-.796l.724-.69c.27.285.52.59.747.91l-.818.576zm.744 1.352a7.08 7.08 0 0 0-.214-.468l.893-.45a7.976 7.976 0 0 1 .45 1.088l-.95.313a7.023 7.023 0 0 0-.179-.483zm.53 2.507a6.991 6.991 0 0 0-.1-1.025l.985-.17c.067.386.106.778.116 1.17l-1 .025zm-.131 1.538c.033-.17.06-.339.081-.51l.993.123a7.957 7.957 0 0 1-.23 1.155l-.964-.267c.046-.165.086-.332.12-.501zm-.952 2.379c.184-.29.346-.594.486-.908l.914.405c-.16.36-.345.706-.555 1.038l-.845-.535zm-.964 1.205c.122-.122.239-.248.35-.378l.758.653a8.073 8.073 0 0 1-.401.432l-.707-.707z"
+              />
+              <path
+                d="M8 1a7 7 0 1 0 4.95 11.95l.707.707A8.001 8.001 0 1 1 8 0v1z"
+              />
+              <path
+                d="M7.5 3a.5.5 0 0 1 .5.5v5.21l3.248 1.856a.5.5 0 0 1-.496.868l-3.5-2A.5.5 0 0 1 7 9V3.5a.5.5 0 0 1 .5-.5z"
+              />
+            </svg>
+            <h3 class="session-title">Introducing Iraq's History</h3>
+            <p class="session-content">
+              Participants get a chance to go back through time and get
+              introduced to the rich culture Iraq and Kurdistan has had and how
+              it can be taken advantage of to move forward.
+            </p>
+          </div>
+          <div class="program-sessions">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="50"
+              height="50"
+              fill="currentColor"
+              class="bi bi-award"
```

Comment:
Good job adding the mid section to your project but it would be nice if you used the actual background image provided in the project design.

This is your design :

![mid section](https://user-images.githubusercontent.com/79506105/198347836-dc6c9ae2-ec7a-4b02-9b4e-60c0e5c3f74a.png)

Please stick to this design:

![original md section](https://user-images.githubusercontent.com/79506105/198347957-928a5cce-e98d-447f-aebb-d755db208b83.png)

You can achieve this by adding the image below and adding a background repeat in your CSS.

Use this image for your background:

![repeat-bg](https://user-images.githubusercontent.com/79506105/198349349-74eb9bcb-553f-497d-bf02-8c86908ed07b.png)